lib/pharos/config_schema.rb

```diff
@@ -113,6 +113,7 @@ def self.messages
           end
           optional(:weave).schema do
             optional(:trusted_subnets).each(type?: String)
+            optional(:no_masq_local).filled(:bool?)
```

Comment:
Would `masq_local` (default `true`) make more sense?

Reply:
As mentioned in the comment, it might actually break an existing cluster.

Reply:
I think you missed my point, `no_masq_local` vs `masq_local` as config variable name.

Reply:
ah, sorry. Now I see what you meant...

I think it makes to sense to keep the option 1-to-1 with the source, i.e. Weave env

Reply:
Ok, fair enough.. I just always feel bad when a boolean variable name starts  with `no` 😄

Reply:
yeah, it's not perfect. but IMO easier to map to downstream configs & docs when needed